tests/acceptance/new-home-test.ts

```diff
@@ -0,0 +1,37 @@
+import { currentURL, visit } from '@ember/test-helpers';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+import { percySnapshot } from 'ember-percy';
+import { module, test } from 'qunit';
+
+import { click, setupOSFApplicationTest } from 'ember-osf-web/tests/helpers';
+
+module('Acceptance | new home page test', hooks => {
+    setupOSFApplicationTest(hooks);
+    setupMirage(hooks);
+
+    test('visiting new-home', async assert => {
+        await visit('/new-home');
+
+        assert.equal(currentURL(), '/new-home', "Still at 'new-home'.");
+        // Check navbar
+        assert.dom('nav.navbar').exists();
+        assert.dom('nav.navbar .service-name').hasText('OSF HOME');
+        assert.dom('nav.navbar .sign-in').exists();
+
+        // Check page
+        assert.dom('[data-test-hero-heading]').hasText('The place to share your research');
```

Comment:
Instead of comparing to static strings, should follow: https://github.com/jamesarosen/ember-i18n/wiki/Doc:-Testing and use the `t` function.